lib/utils/emitDeprecationWarning.mjs

```diff
@@ -2,6 +2,7 @@ import process from 'node:process';
 
 // Each code must be unique.
 const CODES = {
+	RULE: '000', // either removal or renaming
```

Comment:
[suggest] I think using the normally incremented number is fine because `000` looks like a special number. E.g.

```suggestion
	RULE: '006',
```

Reply:
It's the oldest type of deprecation, that's why I chose `000`.
But `006` will do too.